xml/admin_ceph_upgrade.xml

```diff
@@ -624,6 +624,11 @@ exported keyring for client.bootstrap-osd
      </para>
     </listitem>
    </itemizedlist>
+   <para>
+    In addition, before starting the procedure below, you need to upgrade the
+    &sadmin; node to &sls; 12 SP3 and &storage; 5 by running <command>zypper
```

Comment:
We have no sadmin entity. You can either add it to the entity-decl.ent or replace \&sadmin; with \&salt; master.